src/applications/edu-benefits/feedback-tool/actions/schoolSearch.js

```diff
@@ -1,8 +1,9 @@
 import recordEvent from '../../../../platform/monitoring/record-event';
 
-import { trackingPrefix } from '../config/form';
```

Comment:
we really should prohibit default exports

Reply:
what should be used in lieu?